include/zephyr/bluetooth/conn.h

```diff
@@ -1396,6 +1396,20 @@ enum bt_security_err {
 	BT_SECURITY_ERR_UNSPECIFIED,
 };
 
+struct bt_conn_le_cs_procedure_enable_complete {
+	uint8_t config_id;
+	uint8_t state;
```

Comment:
can we have a simple enum for state? Then re-use it here https://github.com/zephyrproject-rtos/zephyr/pull/79183/files#diff-b8bf0043e1d142bd1bc51849856bc2bf6b8e905f25d7bdba9a3d2d934547cce5R639